detection_rules/utils.py

```diff
@@ -33,6 +33,7 @@
 CURR_DIR = os.path.dirname(os.path.abspath(__file__))
 ROOT_DIR = os.path.dirname(CURR_DIR)
 ETC_DIR = os.path.join(ROOT_DIR, "detection_rules", "etc")
+INTEGRATIONS_DIR = os.path.join(ROOT_DIR,"rules","integrations")
```

Comment:
This can be removed as well. Originally we planned to download all of the integration version packages and needed somewhere to store them temporarily but since we can pull the manifest files directly into memory this is no longer needed.